src/pages/Borrow/BorrowHowLong.vue

```diff
@@ -18,31 +20,55 @@
 					preparation, you can get funded very quickly on our platform.
 				</p>
 				<p>
-					Need help getting started? Check out this useful guide.
+					Need help getting started?
+					<a href="https://www.kivaushub.org/borrowers">Check out this useful guide</a>.
 				</p>
 			</div>
 		</div>
 		<!-- eslint-disable max-len -->
 		<kv-button
-			href="https://kivaportal.force.com/portal/secur/CommunitiesSelfRegUi?startURL=%2Fportal%2Fs%2F%3Fref%3Dbam%26refid%3D"
+			:href="`https://kivaportal.force.com/portal/secur/CommunitiesSelfRegUi?startURL=%2Fportal%2Fs%2F%3Fref%3Dbam%26refid%3D${trusteeId}`"
 			class="cta-btn"
 		>
-			I understand
+			I understand, let’s create an account
 		</kv-button>
 		<!-- eslint-enable max-len -->
 	</div>
 </template>
 
 <script>
-import IntroGraphic from '@/assets/inline-svgs/borrow/howlong.svg';
+import gql from 'graphql-tag';
+import _get from 'lodash/get';
 
+import IntroGraphic from '@/assets/inline-svgs/borrow/howlong.svg';
 import KvButton from '@/components/Kv/KvButton';
 
+
 export default {
+	inject: ['apollo'],
 	components: {
 		IntroGraphic,
 		KvButton,
 	},
+	data() {
+		return {
+			trusteeId: ''
+		};
+	},
+	apollo: {
```

Comment:
@emuvente Will this work instead of you passing in a trusteeId from the old stack as a query param? Also wasn't sure of a good way to test this.

Reply:
No, because this is a person who hasn't signed up yet. We're passing along the id of the trustee that referred this user to the /borrow page

Reply:
> No, because this is a person who hasn't signed up yet. We're passing along the id of the trustee that referred this user to the /borrow page

ahh, thanks for the explanation.

Reply:
You should just use the page query variable in the created hook:
```
created() {
    this.trusteeId = _get(this, '$route.query.refid', '');
}
```